ldm/simplet2i.py

```diff
@@ -269,6 +272,11 @@ def process_image(image,seed):
 
         scope = autocast if self.precision == 'autocast' else nullcontext
 
+        if user_sampler is not None:
```

Comment:
This could be refactored to

```python
if user_sampler and (user_sampler != self.sampler_name):
    etc.
```

Reply:
I initially wrote it like that but for some reason the code was ignoring the second check, so I did a separate checker. Let me try again and see.